dashboard/lib/policies/gender.rb

```diff
@@ -1,26 +1,19 @@
 class Policies::Gender
-  OPTIONS = [
-    [nil, ''],
-    ['gender.male', 'm'],
-    ['gender.female', 'f'],
-    ['gender.non_binary', 'n'],
-    ['gender.not_listed', 'o'],
-    ['gender.none', '-'],
-  ].freeze
+  MALE_REGEX = /(^m$|\bmale|boy|guy|\bm(a|e)n|masculin|him|\bhe\b|hombre|dude|\bmail)/
+  FEMALE_REGEX = /(^f$|\bfemal|girl|gal|\bwom(a|e)n|fem(e|i)nin|she|\bher|mujer|\bfemail|\bfem)/
+  NON_BINARY_REGEX = /(^n$|they|them|non((\ |-)?)binary|inter(\ )?sex|gender(\ )?fluid|inter(\ )?gender|agender|boyflux|\btrans|\bit)/
 
   def self.normalize(gender)
     return nil if gender.blank?
-    case gender.downcase
-    when 'f', 'female'
-      'f'
-    when 'm', 'male'
-      'm'
-    when 'o', 'notlisted'
-      'o'
-    when 'n', 'nonbinary', 'non-binary'
-      'n'
+    lowercase_gender = gender.downcase
+    if NON_BINARY_REGEX.match(lowercase_gender)
+      return 'n'
+    elsif MALE_REGEX.match(lowercase_gender)
+      return 'm'
+    elsif FEMALE_REGEX.match(lowercase_gender)
+      return 'f'
     else
-      nil
+      return 'o'
```

Comment:
This was probably already brought up with the RED team, but I just wanted to point out in the PR that "none" or "Prefer not to answer" is not longer something we now probably be classified as "not_listed"

Reply:
Yep this was brought up in the last meeting we had with product and RED